lib/adrian/directory_queue.rb

```diff
@@ -52,6 +51,10 @@ def include?(value)
       items.include?(item)
     end
 
+    def items_length
```

Comment:
this was added for testing purposes and returns the length of the internal `@items` list, not `#items`. so it needs to be public, but the name could definitely be better. maybe `internal_items_count`?